<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Difficulty Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSlider" name="zoomLevel">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="box">
      <property name="title">
       <string>Blockchain Graphs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="GraphWidget" name="graphsFrame" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Block:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="blockIndex">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Algo">
           <property name="text">
            <string>Satoshi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="maximum">
            <number>2016</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="minersForm">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>711</width>
         <height>68</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>S&amp;imulation</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Pause"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
    <widget class="QMenu" name="menuOld">
     <property name="title">
      <string>Old</string>
     </property>
     <addaction name="actionDeadalnix"/>
     <addaction name="actiondEDA"/>
     <addaction name="actiondEDAmodTom"/>
    </widget>
    <addaction name="menuOld"/>
    <addaction name="actionSatoshi"/>
    <addaction name="actionEDA"/>
    <addaction name="actionNeil"/>
    <addaction name="actioncw144"/>
    <addaction name="actionwt144"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Miner"/>
   <addaction name="action_Pause"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Pause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_Miner">
   <property name="text">
    <string>New Miner</string>
   </property>
   <property name="toolTip">
    <string>Append new miner</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEDA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;EDA (deadalnix)</string>
   </property>
   <property name="toolTip">
    <string>Enable the Emergency Difficulty Adjustment</string>
   </property>
  </action>
  <action name="actionNeil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Neil</string>
   </property>
   <property name="toolTip">
    <string>Select the new Algorithm</string>
   </property>
  </action>
  <action name="actionSatoshi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Satoshi</string>
   </property>
   <property name="toolTip">
    <string>Select the original algorithm</string>
   </property>
  </action>
  <action name="actiondEDA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;dEDA </string>
   </property>
   <property name="toolTip">
    <string>Select dualEDAv1</string>
   </property>
  </action>
  <action name="actiondEDAmodTom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>dEDA&amp;modTom </string>
   </property>
   <property name="toolTip">
    <string>Select dualEDAmodTom</string>
   </property>
  </action>
  <action name="actionDeadalnix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deada&amp;lnix </string>
   </property>
   <property name="toolTip">
    <string>Select Deadalnix</string>
   </property>
  </action>
  <action name="actioncw144">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;cw144  (deadalnix)</string>
   </property>
   <property name="toolTip">
    <string>Select cw144</string>
   </property>
  </action>
  <action name="actionwt144">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;wt144 </string>
   </property>
   <property name="toolTip">
    <string>Select wt144</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>graphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
